### PR TITLE
new chapter for vector tiles

### LINKED ISSUE #6086: UI Improvements for vector tile symbology/labeling (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#38654
Author: @nyalldawson
QGIS version: 3.16

**UI Improvements for vector tile symbology/labeling**

### PR Description:
Some quality of life improvements for users editing complex vector tiles styles:

- show tooltips in the lists, so you can easily see the full filter rules and names without resizing columns
- show the current canvas zoom level in the widgets, and allow users to filter the list of styles to only show those which are currently visible. Makes it much easier to identify which style is being used to render the current map!
- Set the @zoom_level variable to the correct canvas zoom level when editing symbols, so that expression previews work nicely.

### Commits tagged with [need-docs] or [FEATURE]
"[feature][vectortiles] Add checkbox for showing only visible rules\nin the vector tiles labeling and renderer widgets\n\nMakes it much easier to locate troublesome rules or work with\ncomplex vector tile styling"

### PR DESCRIPTION
**Pull request by lajunek**
Goal: The user shall get compact information about vector tiles and their usage in QGIS.

Ticket(s): 

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Supported Formats
+=================
+
+There is support for vector tiles through
+
+    * remote sources (HTTP/S) - with XYZ template - ``type=xyz&url=http://example.com/{z}/{x}/{y}.pbf``
+    * local files - with XYZ template - e.g. ``type=xyz&url=file:///path/to/tiles/{z}/{x}/{y}.pbf``
+    * local MBTiles database - e.g. ``type=mbtiles&url=file:///path/to/file.mbtiles``
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst:57]:
```suggestion
There is support for vector tiles through:

* remote sources (HTTP/S) - with XYZ template - ``type=xyz&url=http://example.com/{z}/{x}/{y}.pbf``
* local files - with XYZ template - e.g. ``type=xyz&url=file:///path/to/tiles/{z}/{x}/{y}.pbf``
* local MBTiles database - e.g. ``type=mbtiles&url=file:///path/to/file.mbtiles``
```
No Indentation

File: docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Working with Vector Tiles
+***********************
+
+.. only:: html
+
+.. contents::
+
+   :local:
+
+What are Vector Tiles?
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst:24]: Thanks for this section. :+1:

File: docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+4
+
+Working with Vector Tiles
+***********************
+
+.. only:: html
+
+.. contents::
+
+   :local:
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst:22]: Should be indented to display the table of contents
```suggestion
   .. contents::

      :local:
```

File: docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+​
+
+3
+
+***********************
+
+4
+
+Working with Vector Tiles
+***********************
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst:16]: Should be longer, I think

**Review by DelazJ — COMMENTED** [review]
Thanks @lajunek for handing this.

File: docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst (hunk 4 above)
Comment (lajunek) [reply docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst]: @DelazJ can I merge after fixing the HTML build?

Comment (DelazJ) [reply docs/user_manual/working_with_vector_tiles/vector_tiles_properties.rst]: If you want. This PR provides general information about tile vector and unfortunately I'm not the best person to review the contents.
But if you want to merge, then I'd suggest to remove the `fix #6086` from your initial message. Afaict, that issue report provides information on vector tile symbology and labeling properties dialog and these properties are not explored in this PR. That issue report should not be closed, yet.

**Comment by lajunek** [comment]
more information about vector tiles usage are coming soon

**Pull request merged**